src/pyOpenMS/pyopenms/_dataframes.py

```
@@ -341,601 +54,58 @@ def get_df(self, columns: Union[None, List[str]] = None):
-PeakMap.__name__ = 'PeakMap'
-
-MSExperiment = _MSExperimentDF
-MSExperiment.__module__ = _MSExperiment.__module__
-MSExperiment.__name__ = 'MSExperiment'
-
-
-# TODO think about the best way for such top-level function. IMHO in python, encapsulation in a stateless class in unnecessary.
-#   We should probably not just import this whole submodule without prefix.
-def peptide_identifications_to_df(peps: _PeptideIdentificationList, decode_ontology : bool = True,
+def peptide_identifications_to_df(peps: _PeptideIdentificationList, decode_ontology: bool = True,
                                   default_missing_values: dict = None,
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)